admin-dev/themes/new-theme/js/pages/product/mixins/is-attribute-selected.ts

```diff
@@ -22,10 +22,9 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-import Vue from 'vue';
-import {Attribute, AttributeGroup} from '@pages/product/types';
+import {Attribute, AttributeGroup} from '@pages/product/components/generator';
```

Comment:
Following my previous comment, I think it's cleaner to have an independent file containing the types that you can import from Instead of importing them from the application index/main file